lms/djangoapps/course_api/serializers.py

```diff
@@ -73,10 +75,24 @@ class CourseSerializer(serializers.Serializer):  # pylint: disable=abstract-meth
     pacing = serializers.CharField()
     mobile_available = serializers.BooleanField()
     hidden = serializers.SerializerMethodField()
+    course_modes = serializers.SerializerMethodField()
```

Comment:
hmmm, locally we just set-up "Courses API URL" in partner. I think, we have to set-up "E-Commerce API URL" for that. Perhaps, just a configuration issue.